llvm/test/lab2/veselov_mikhail/test.ll

```diff
@@ -0,0 +1,127 @@
+; RUN: opt -load-pass-plugin=%llvmshlibdir/VeselovBitwiseShiftMultiplier%shlibext -passes=vesel-mult-shift -S %s | FileCheck %s
```

Comment:
Could you please also test the following case:
void f5(){
    int a = 4;
    int c = a * 3;
}